src/sentry/static/sentry/app/views/alerts/details/body.tsx

```diff
@@ -213,6 +228,45 @@ export default class DetailsBody extends React.Component<Props> {
                   data={stats.eventStats.data}
                   detected={incident.dateDetected}
                   closed={incident.dateClosed}
+                  warningMarkLine={
```

Comment:
Maybe it would make sense to move this actually into the incidents chat component? And we just pass the incident itself in?